fabfile.py

```diff
@@ -142,6 +151,9 @@ def install():
         run('mkdir media')
         run('chown www-data:www-data media')
 
+    # Export Sentry DSN key
+    run('echo "export VOLONTULO_SENTRY_DSN='{}'.format('VOLONTULO_SENTRY_DSN') >> ~/.bash_profile')
```

Comment:
that's will raise syntax error and this variable need to be pushed also to `uwsgi.ini`